<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialGeneratorDialog</class>
 <widget class="QDialog" name="SerialGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Generator Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelProtoSerial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prototype serial number:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrototypeSerial">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>????????</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelReplaceWith">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>52</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace wildcards (?) with:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDigits">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Digits</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUppercase">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Uppercase letters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLowercase">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lowercase letters</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxExportOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Export Options</string>
   </property>
   <widget class="QCheckBox" name="checkBoxExportToFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to configuration file</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxEnable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable generator by default</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOverwrite">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Overwrite default serial number</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SerialGeneratorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SerialGeneratorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
